steps/qaoa.py

```diff
@@ -55,3 +59,10 @@ def get_maxcut_hamiltonian(graph, scaling=1.0, shifted=False):
         graph_object, scaling=scaling, shifted=shifted
     )
     save_qubit_operator(hamiltonian, "hamiltonian.json")
+
+
+def convert_relaxed_solution_to_angles(solution, epsilon=0.5, decimals=8):
+    solution = np.around(load_list(solution), decimals=decimals)
```

Comment:
How come we're rounding these values?

Reply:
Follow-up, is this the right place to round these numbers? Would it make more sense for them to be rounded inside of `convert_relaxed_solution_to_angles`?